CMakeLists.txt

```diff
@@ -318,6 +318,8 @@ if (MSVC)
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 	add_definitions(-Wno-c++11-extensions)
 	add_definitions(-fcolor-diagnostics)
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I have this PR to add C++11 support to the CMake file. if it looks good to you, I'll merge it: https://github.com/arvidn/libtorrent/pull/682